includes/class-satispress-package-plugin.php

```diff
@@ -218,9 +218,72 @@ public function get_version() {
 	 */
 	protected function get_data( $prop ) {
 		if ( empty( $this->data ) ) {
-			$this->data = get_plugin_data( $this->get_file(), false, false );
+			$this->data = $this->get_plugin_data( $this->get_file(), false, false );
 		}
 
 		return ( isset( $this->data[ $prop ] ) ) ? $this->data[ $prop ] : '';
 	}
+
+
+	/**
+	 * Get the plugin details
+	 *
+	 * This function is a copy of the original WordPress core function, with "Plugin Type"
+	 * Added for distinction wordpress-plugin and wordpress-muplugin
+	 *
+	 * @since 0.2.2
+	 *
+	 * @param	string		$plugin_file	Path to the plugin file.
+	 * @param	boolean		$markup		 	If the returned data should have HTML markup applied.
+	 * @param	boolean		$translate		If the returned data should be translated.
+	 *
+	 * @return  boolean		$markup
+	 */
+	protected function get_plugin_data( $plugin_file, $markup = true, $translate = true ) {
+
+	    $default_headers = array(
+			'Name' => 'Plugin Name',
+	        'PluginURI' => 'Plugin URI',
+	        'Version' => 'Version',
+	        'Description' => 'Description',
+	        'Author' => 'Author',
+	        'AuthorURI' => 'Author URI',
+	        'TextDomain' => 'Text Domain',
+	        'DomainPath' => 'Domain Path',
+			'Type' => 'Plugin Type',
```

Comment:
Rather than copy `get_plugin_data()` from core, custom headers can be whitelisted using the `extra_plugin_headers` [filter](https://github.com/WordPress/WordPress/blob/3821ff74bcf9c960f8a7f1fae452d207135ecc64/wp-includes/functions.php#L4805).